src/frontends/tensorflow_common/src/op/div.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_op_table.hpp"
+#include "openvino/op/divide.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+OutputVector translate_div_op(const NodeContext& node) {
+    default_op_checks(node, 2, {"Div"});
+    auto x = node.get_input(0);
+    auto y = node.get_input(1);
+    bool m_pythondiv = false;
+
+    // Check if the element type is a integer
+    if (x.get_element_type().is_integral_number()) {
```

Comment:
please add additional check `x.get_element_type().is_signed()`. For unsigned, we should not have a problem